# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/bison-30-warn_redefined.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
warn symbol is part of libc.o, then i have to look for a
different name
  Alejandro

--- ./src/complain.c.orig	2004-02-11 19:14:26.000000000 -0300
+++ ./src/complain.c	2004-02-11 19:16:19.000000000 -0300
@@ -87,7 +87,7 @@
 }
 
 void
-warn (const char *message, ...)
+warn_this (const char *message, ...)
 {
   va_list args;
 
--- ./src/complain.h.orig	2004-02-11 19:14:33.000000000 -0300
+++ ./src/complain.h	2004-02-11 19:16:32.000000000 -0300
@@ -27,7 +27,7 @@
 
 /* Informative messages, but we proceed.  */
 
-void warn (char const *format, ...)
+void warn_this (char const *format, ...)
   __attribute__ ((__format__ (__printf__, 1, 2)));
 
 void warn_at (location loc, char const *format, ...)
--- ./src/files.c.orig	2004-02-11 19:14:43.000000000 -0300
+++ ./src/files.c	2004-02-11 19:16:41.000000000 -0300
@@ -333,5 +333,5 @@
   for (j = 0; j < names; j++)
     for (i = 0; i < j; i++)
       if (strcmp (name[i], name[j]) == 0)
-	warn (_("conflicting outputs to file %s"), quote (name[i]));
+	warn_this (_("conflicting outputs to file %s"), quote (name[i]));
 }
